src/openpersonen/api/utils.py

```diff
@@ -7,3 +12,30 @@ def convert_empty_instances(dictionary):
             dictionary[key] = None
         elif isinstance(dictionary[key], dict):
             convert_empty_instances(dictionary[key])
+
+
+def calculate_age(person_birth_date):
+
+    try:
+        return relativedelta(
+            datetime.now(),
+            datetime.strptime(person_birth_date, "%Y%m%d"),
+        ).years
+    except ValueError:
+        try:
+            return datetime.now().year - int(
+                person_birth_date[settings.YEAR_START : settings.YEAR_END]
+            )
+        except ValueError:
+            return 0
+
+
+def is_valid_date_format(date):
+    if len(date) != 8:
+        return False
+
+    try:
+        int(date)
+        return True
+    except ValueError:
+        return False
```

Comment:
I think for this it would be easier to use `.isdigit()`:
```suggestion
    return date.isdigit()
```

Reply:
Agreed!